src/components/Party.tsx

```diff
@@ -50,6 +50,7 @@ export const Party = ({ username, roomId }: PartyProps) => {
               className="border border-gray-400 rounded p-2"
               name="message"
               type="text"
+              required
```

Comment:
I'm not sure if this is the desired approach. Usually, we are used to a different chat experience. Perhaps disabling the 'Send Message' button or preventing message posting when pressing 'Enter' would be preferable?"

Reply:
Sorry @Arol15, I didn't notice it was a draft. 

I feel it's fine with the current solution about the blank input.

Reply:
Required is all good. It'll do the job. 🫡